### ISSUE TITLE
How to improve data simulation

### ISSUE THREAD

**Issue by anderdnavarro** [Team-TUD/CTAB-GAN-Plus#13]
Hi @zhao-zilong,

I was wondering if you have any suggestion about how to improve this simulation. When I train the model (1 col x 22928 rows), the simulation follows the same density distribution:

![Density](https://github.com/Team-TUD/CTAB-GAN-Plus/assets/33023846/137f04b2-38e7-4d7b-8138-8cfc27ad7105)

But when I group by bins to see how close the simulated data is to the original, the histogram is very different:
Original
![Original_hist](https://github.com/Team-TUD/CTAB-GAN-Plus/assets/33023846/aa590746-b2bf-42ab-a7c6-2ce4276b2848)
Simulation
![Sim_hist](https://github.com/Team-TUD/CTAB-GAN-Plus/assets/33023846/b64835cc-f57c-4c1f-bde7-43c05eae5747)

I have played around with the loss function a bit and transforming the training data, but without any improvement. Maybe it is not possible to achieve this level of precision, although I thought it was possible looking at the plots that are available in the paper for the _hours-per-week_ dataset.

I tried to take a look at the loss values and above epoch 35 stabilizes at around 20.

Thank you very much!!
Ander

**Comment by zhao-zilong** [comment]
Hi @anderdnavarro 

Sorry that I somehow missed this question, I think it is difficult to achieve that level of precision. But I would suggest you to try our new algorithm: https://github.com/zhao-zilong/Tabula , I would say in the fidelity part, it can achieve even better results than ctabgan+, but of course, it takes more time to train than ctabgan+.

Best,

Zilong

**Comment by anderdnavarro** [comment]
Hi @zhao-zilong,

No worries! Thanks for the suggestion! I will read your new paper to see if it can help me.

Best,
Ander
